src/promptflow/promptflow/azure/_storage/cosmosdb/span.py

```diff
@@ -0,0 +1,59 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Any, Dict
+
+from promptflow._constants import SpanFieldName
+from promptflow._sdk.entities._trace import Span as SpanEntity
+
+
+class Span:
+    __container__ = "Span"
+
+    name: str = None
+    context: dict = None
+    kind: str = None
+    parent_id: str = None
+    start_time: str = None
+    end_time: str = None
+    status: dict = None
+    attributes: dict = None
+    events: list = None
+    links: list = None
+    resource: dict = None
+    id: str = None
+    partition_key: str = None
+
+    def __init__(self, span: SpanEntity) -> None:
+        self.name = span.name
+        self.context = span._content[SpanFieldName.CONTEXT]
+        self.kind = span._content[SpanFieldName.KIND]
+        self.parent_id = span.parent_span_id
+        self.start_time = span._content[SpanFieldName.START_TIME]
+        self.end_time = span._content[SpanFieldName.END_TIME]
+        self.status = span._content[SpanFieldName.STATUS]
+        self.attributes = span._content[SpanFieldName.ATTRIBUTES]
+        self.events = span._content[SpanFieldName.EVENTS]
+        self.links = span._content[SpanFieldName.LINKS]
+        self.resource = span._content[SpanFieldName.RESOURCE]
+        self.partition_key = span.session_id
+        self.id = span.span_id
+
+    def persist(self, client):
+        if self.id is None or self.partition_key is None or self.resource is None:
+            return
+
+        resource_attributes = self.resource.get("attributes", None)
```

Comment:
we already defined some consts for attributes name, try to replace this hard code to that one

Reply:
Fixed. Thanks.